10up/animation-best-practices
# Feature: Vector line drawing

### Description of the Change


Adds example for vector line drawing.




### How to test the Change

1. Checkout branch
2. Run `npm install` and `npm run build`
3. Check dist of example in a browser

### Changelog Entry

- Added - Vector line drawing example in CSS, WAAPI, and GSAP


### Credits


@Firestorm980 

### Checklist:
